install.sh

```diff
@@ -36,18 +36,20 @@ test -w "$HOME/$CONFIG_FILE" &&
   cp -a "$HOME/$CONFIG_FILE" "$HOME/$CONFIG_FILE.bak" &&
   echo -e "\033[0;32mYour original $CONFIG_FILE has been backed up to $CONFIG_FILE.bak\033[0m"
 
-cp "$HOME/.bash_it/template/bash_profile.template.bash" "$HOME/$CONFIG_FILE"
+cp "$BASH_IT/template/bash_profile.template.bash" "$HOME/$CONFIG_FILE"
+sed -i "s|{{BASH_IT}}|$BASH_IT|" "$HOME/$CONFIG_FILE"
+exit
 
 echo -e "\033[0;32mCopied the template $CONFIG_FILE into ~/$CONFIG_FILE, edit this file to customize bash-it\033[0m"
 
 function load_one() {
   file_type=$1
   file_to_enable=$2
-  [ ! -d "$BASH_IT/$file_type/enabled" ] && mkdir "$BASH_IT/${file_type}/enabled"
+  mkdir -p "$BASH_IT/${file_type}/enabled"
```

Comment:
I belive this should be replaced with:

`[ ! -d "$BASH_IT/$file_type/enabled" ] && mkdir -p "$BASH_IT/${file_type}/enabled"`

This will make sure `mkdir` is NOT run if directory already exists.